en/docs/usecases/integration-service-as-a-managed-api/how-to-expose-an-integration-service-as-a-managed-api.md

```diff
@@ -0,0 +1,176 @@
+# How to Expose an Integration Service as a Managed API
+
+## What you'll build
+
+In this tutorial, you'll define an integration service using WSO2 Integrator: BI and expose it as a managed API to the API marketplace. API consumers then **discover** the API from the marketplace, **subscribe** to it, and **use it** for application development.
+
+<a href="{{base_path}}/assets/usecases/integration-service-as-a-managed-api/exposing-service-as-managed-api.png"><img src="{{base_path}}/assets/usecases/integration-service-as-a-managed-api/exposing-service-as-managed-api.png" alt="exposing integration service as a managed api"></a>
```

Comment:
This is not getting rendered properly

<img width="1122" height="285" alt="Image" src="https://github.com/user-attachments/assets/2cc838e5-5129-476b-8738-b81bab759645" />